downstairs/src/main.rs

```diff
@@ -182,6 +182,45 @@ enum Args {
         bind_addr: SocketAddr,
     },
     Version,
+    /*
```

Comment:
If this is a doc comment (`/// Measure an isolated downstairs`), it shows up in clap's help:
```
Usage: crucible-downstairs <COMMAND>

Commands:
  create
  dump
  export
  run
  repair-api
  serve
  version
  dynamometer  Measure an isolated downstairs
  help         Print this message or the help of the given subcommand(s)
```

Reply:
see 3cb447e